lib/ip_record.js

```diff
@@ -123,6 +124,32 @@ module.exports = function (limits, now) {
     this.as = this._trim(now, this.as, limits.maxAccountStatusCheck)
   }
 
+  IpRecord.prototype.isOverSMSLimit = function () {
+    this.trimSMSRequests(now())
+    // Limit based on number of unique sms request sent by this IP
+    var count = 0
+    var seen = {}
+    this.sms.forEach(function(info) {
+      if (!(info.u in seen)) {
```

Comment:
I can see they're used elsewhere here but, for the record, I'm not a fan of these one-character (and two-character) property names. It's like we're explicitly trying to make the code harder to follow. Using `t` instead of `time` at least has an element of logic, but why `u` for the phone number? Am I missing something?